vfp/WVFPDP-01.DATA

```diff
@@ -0,0 +1,333 @@
+-- In this section simulation grid parameters are edited
+-- ***************************************************
+
+EDIT
+
+-- ***************************************************
+
+
+-- ***************************************************
+-- In this section fluid-rock properties and 
+-- relative permabilities are given
+-- ***************************************************
+
+PROPS
+
+-- ***************************************************
+
+INCLUDE
+ './include/sgof.txt' /
+
+
+INCLUDE
+ './include/swof.txt' /
+
+-- ---------------------------------------------------
+
+-- Include PVT data
+INCLUDE
+  './include/example_pvt.txt' /
+
+-- ***********************************************************
+-- In this section simulation grid region parameters are given
+-- ***********************************************************
+
+REGIONS
+
+-- ***************************************************
+--
+-- ***************************************************
+-- In this section the initialization parameters aand
+-- dynamic parameters are defined
+-- ***************************************************
+
+EQUALS
+ 'EQLNUM' 1 /
+ 'SATNUM' 1 /
+/
+ 
+SOLUTION
+
+-- ***************************************************
+
+--
+--Simulation model initialisation data
+--
+--   DATUM  DATUM   OWC     OWC    GOC    GOC    RSVD   RVVD   SOLN
+--   Depth  Pres.   Depth   Pcow   Depth  Pcog   Table  Table  Method
+EQUIL
+     2020.0   300.0   2030.0  0.0    500    0.0     1     1      0 / 
+    
+
+--
+-- ---------------------------------------------------
+-- Dissolved gas-oil ratio versus depth, 
+
+RSVD
+
+ 1500 100.0
+ 4000 100.0  /
+ 
+
+RPTSOL
+  'THPRES' /
+
+
+RPTRST
+ 'BASIC=2'  'PBPD' /
+ 
+
+AQUFETP
+---- Aqu        depth          Pr       vol       Comp           PI     PVTW
+   1           2020.0        300.0    5.0E+9      1.5312E-4    995      1*    /
+/
+
+
+AQUANCON
+--Aqu  I1   I2   J1   J2   K1   K2   face   influx coef.   influx coef. multiplier
+    1  6    6     1    4    4    7    'I+'        1*               1*  /
+/
+
+
+--
+-- **************************************************************************************
+-- In this section simulation output data to be written to summary file are defined
+-- **************************************************************************************
+
+SUMMARY
+
+-- ***************************************************
+-- ---------------------------------------------------
+-- Summary data to be written to summary file
+--
+--
+--
+-- **************************************************************************************
+-- In this section data required to describe history and prediction is given
+-- - well completions, well production/injection, well constraints
+-- - platform/production unit constraints, etc.
+-- **************************************************************************************
+
+WOPR
+ /
+
+WBHP
+ /
+ 
+WTHP
+ /
+
+WGOR
+ /  
+
+WWCT
+/
+
+WLPR
+ /
+
+WGIR
+/
+ 
+FPR
+
```

Comment:
Would it make sense to add a few aquifer-related vectors too, e.g., `AAQP` and/or `AAQR`?

Reply:
Yes, good point. I've also added gas lift injection rate since this is also used for the producer.